extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/UserRoleListService.java

```diff
@@ -104,14 +110,15 @@ public RoleListWrapper getAllRoles() {
     }
 
     existingRoles.addAll( allRoles );
-
+    if ( excludeAnonymous && getAnonymousRole() != null ) {
+      existingRoles.remove( getAnonymousRole() );
+    }
```

Comment:
Please surround this new block with an empty line, before and after.

Reply:
Addressed